www/artists/emmalake/artists.html

```diff
@@ -27,66 +27,75 @@ <h1>
                         Workshop Leaders by Date:
                     </p>
                     <div class="c1">
-                        1954-1955 Beginnings: <a class="a" href="../amckay/index.html">Arthur McKay</a>,<a class="a" href="../klochhead/index.html">Kenneth Lochhead</a>
+                        1954-1955 Beginnings: <a class="b" href="https://saskartists.ca/artists/amckay/index.html">Arthur McKay</a>,<a class="b" href="https://saskartists.ca/artists/klochhead/index.html">Kenneth Lochhead</a>
```

Comment:
Absolute links to SaskArtists are not allowed. This is to avoid issues if we change our domain at a later date.